src/App.vue

```diff
@@ -765,6 +765,23 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
 
     this.setIsSaving(false)
   }
+
+  /**
+   * Helper to show the confirm dialogs.
+   * @param title The title content in dialog header
+   * @param message The content body
+   * @param yes The YES action label
+   * @param no The NO cancel label
+   * */
+  async showConfirmDialog (title: string, message: string, yes: string, no: string = null): Promise<any> {
+    return this.$refs.confirm.open(title, message, {
+      width: '45rem',
+      persistent: true,
+      yes: yes,
+      no: null,
+      cancel: no
```

Comment:
Do you think it makes more sense to make no=no and leave cancel for the case where the user dismisses the dialog? (I think Esc does work to dismiss the dialog, and will reject the promise, ie, throw an exception here.)

Reply:
Sorry, absolutely. this is was a mistake when i was reverting some testing i was doing with regards to globalizing the Confirm Dialog.

Reply:
Fixed!